docs/components/text-inputs.md

```diff
@@ -140,7 +140,7 @@ variations:
               title="Test input"
               class="a-text-input">
           <button class="a-btn a-btn__link">
-              <img src="/design-system/assets/icons/error.svg" class="cf-icon-svg" />
+              {% include icons/error.svg %}
```

Comment:
since this will now be the code in the HTML code snippet:
```
<a class="a-link a-link__icon" href="#">
    <span class="a-link_text">Example with icon</span>
    {% include icons/download.svg %}
</a>
```

is that ok? is this copy and pasteable like this to cfgov-refresh or some other site using capital-framework?

Reply:
Correct, it's the same format used in the current CF docs:

<img width="585" alt="Screen Shot 2019-07-30 at 2 54 51 PM" src="https://user-images.githubusercontent.com/1060248/62157008-2a12ca80-b2da-11e9-98d5-9e21b9ce0e93.png">